src/DatabaseConfig.cs

```diff
@@ -19,9 +19,11 @@ public class DatabaseConfig
     public string DatabaseName { get; set; }
 
     /// <summary>
-    ///   If this isn't supplied we build one from the database default backup path.
-    ///   See <see cref="Server.DefaultBackupPathTemplate" />
+    ///   SQL query text to retrieve the desired backup path location. If none is provided, Server.DefaultBackupPathTemplate is
+    ///   used.   See <see cref="Server.DefaultBackupPathTemplate" />
+    ///   <example> SELECT '\\my\backup\location\'</example>
+    ///   <example> SELECT Backup_path FROM [msdb].[dbo].[_Sys_Backup_config]</example>
     /// </summary>
-    public string BackupPathTemplate { get; set; }
+    public string BackupPathSqlQuery { get; set; } = "SELECT Backup_path FROM [msdb].[dbo].[_Sys_Backup_config]";

```

Comment:
Right. The "note" in my PR comment above says this should probably be removed. I left it in as an example so that we understand what I'm trying to do for CRDS here.
The default, empty string, would be to execute the query you typed above. So you can leave the string empty instead of setting that string as the default